hrqb/tasks/sql/employee_appointments.sql

```diff
@@ -11,6 +11,7 @@ CHANGELOG
     - 2024-05-13 Query created and added
     - 2024-06-03 Limit rows to appointments that end on or before 2019-01-01
     - 2024-06-05 Do not filter on benefits types
+    - 2025-02-05 Remove 2019-01-01 date cutoff entirely
```

Comment:
Hmm, curious why you included this in the changelog if no changes -- beyond this updated comment -- are present in this script?

Reply:
Maybe it got lost in the diff, but this was removed in this file:

```sql
where a.APPT_END_DATE >= TO_DATE('2019-01-01', 'YYYY-MM-DD')
```